### PR TITLE
Install the Service Points + Order Management feature

### PR DESCRIPTION
**Pull request by michbeck**
## PR Description

- Epic Ticket: https://spryker.atlassian.net/browse/CC-26584
- IG Ticket: https://spryker.atlassian.net/browse/CC-30055

### CODE REVIEW

**Review by AlexSlawinski — DISMISSED** [review]

**Comment by AlexSlawinski** [comment]
Approved, good from tech writing perspective @michbeck

**Comment by michbeck** [comment]
Please note: `cms_block.csv` appended in [CC-30055](https://spryker.atlassian.net/browse/CC-30055) needs to uploaded to have up to date one mentioned in [File details - cms_block.csv](https://docs.spryker.com/docs/pbc/all/content-management-system/202307.0/base-shop/import-and-export-data/file-details-cms-block.csv.html).

File: _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md [hunk 1]
```diff
@@ -10,9 +10,12 @@ Follow the steps below to install the Service Points feature core.
...
+| Spryker Core                | {{page.version}} | [Install the Spryker Core feature](/docs/pbc/all/miscellaneous/{{page.version}}/install-and-upgrade/install-features/install-the-spryker-core-feature.html)                                                  |
+| Shipment                    | {{page.version}} | [Shipment feature integration](/docs/pbc/all/carrier-management/{{page.version}}/unified-commerce/enhanced-click-and-collect/install-and-upgrade/install-the-shipment-feature.html)                          |
+| Customer Account Management | {{page.version}} | [Customer account management feature integration](/docs/pbc/all/customer-relationship-management/{{page.version}}/install-and-upgrade/install-features/install-the-customer-account-management-feature.html) |
+| File details CMS block      | {{page.version}} | [File details - cms_block.csv](/docs/pbc/all/content-management-system/{{page.version}}/base-shop/import-and-export-data/file-details-cms-block.csv.html#import-file-command)                                |
```
Comment (dmiseev) [inline _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md:18]: Service Points feature does not depend on these features.
Revert table.

File: _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md [hunk 2]
```diff
@@ -656,6 +674,53 @@ class PublisherDependencyProvider extends SprykerPublisherDependencyProvider
 }
 ```
 
+#### Register sales plugins
```
Comment (dmiseev) [inline _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md:677]: Please move adjustments below to separate IG - `Service Points + Order Management feature`

Comment (michbeck) [reply _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md:677]: And transfers, modules above not? Just wondering while plugins depend on these. 🤔

Comment (dmiseev) [reply _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md:677]: no, modules and transfers should be located in the main feature.

File: _includes/pbc/all/install-features/202400.0/install-the-service-points-order-management-feature.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+Follow the steps below to install the Service Points + Order Management feature.
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME             | VERSION          | INTEGRATION GUIDE                                                                                                                                                                       |
+|------------------|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Service Points   | {{page.version}} | [Install the Service Points feature](/docs/pbc/all/service-points/{{page.version}}/install-and-upgrade/install-the-service-points-feature.html)                                         |
```
Comment (dmiseev) [inline _includes/pbc/all/install-features/202400.0/install-the-service-points-order-management-feature.md:15]: Add also Order management dep.

File: _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md [hunk 4]
```diff
@@ -10,9 +10,9 @@ Follow the steps below to install the Service Points feature core.
...
-|--------------|------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Spryker Core | {{page.version}} | [Install the Spryker Core feature](/docs/pbc/all/miscellaneous/{{page.version}}/install-and-upgrade/install-features/install-the-spryker-core-feature.html) |
+| NAME                        | VERSION          | INTEGRATION GUIDE                                                                                                                                           |
+|-----------------------------|------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Spryker Core                | {{page.version}} | [Install the Spryker Core feature](/docs/pbc/all/miscellaneous/{{page.version}}/install-and-upgrade/install-features/install-the-spryker-core-feature.html) |
```
Comment (dmiseev) [inline _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md:15]: Pease revert the changes.

File: _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md [hunk 5]
```diff
@@ -24,26 +24,29 @@ composer require spryker-feature/service-points: "{{page.version}}" --update-wit
...
 ## 2) Set up database schema and transfer objects
 
 1. Adjust the schema definition so entity changes trigger events.
 
-| AFFECTED ENTITY               | TRIGGERED EVENTS                                                                                                              |
-|-------------------------------|-------------------------------------------------------------------------------------------------------------------------------|
-| spy_service_point             | Entity.spy_service_point.create<br>Entity.spy_service_point.update<br>Entity.spy_service_point.delete                         |
-| spy_service_point_address     | Entity.spy_service_point_address.create<br>Entity.spy_service_point_address.update<br>Entity.spy_service_point_address.delete |
-| spy_service_point_store       | Entity.spy_service_point_store.create<br>Entity.spy_service_point_store.update<br>Entity.spy_service_point_store.delete       |
+| AFFECTED ENTITY           | TRIGGERED EVENTS                                                                                                              |
```
Comment (dmiseev) [inline _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md:45]: Revert changes.

File: _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md [hunk 6]
```diff
@@ -1371,9 +1385,9 @@ class ShopApplicationDependencyProvider extends SprykerShopApplicationDependency
...
-|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| ServicePointSearchWidget | `{% raw %}{%{% endraw %} widget 'ServicePointSearchWidget' args [...] only {% raw %}%}{% endraw %}{% raw %}{%{% endraw %} endwidget {% raw %}%}{% endraw %}` |
+| WIDGET                                      | VERIFICATION                                                                                                                                                                    |
+|---------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ServicePointSearchWidget                    | `{% raw %}{%{% endraw %} widget 'ServicePointSearchWidget' args [...] only {% raw %}%}{% endraw %}{% raw %}{%{% endraw %} endwidget {% raw %}%}{% endraw %}`                    |
```
Comment (dmiseev) [inline _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md:1390]: Revert changes.

**Review by dmiseev — APPROVED** [review]

**Review by philinecao — APPROVED** [review]

File: _includes/pbc/all/install-features/202400.0/install-the-service-points-order-management-feature.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+| Order Management | {{page.version}} | [Order Management feature integration](/docs/pbc/all/order-management-system/{{page.version}}/base-shop/install-and-upgrade/install-features/install-the-order-management-feature.html) |
+
+## 1) Set up behavior
```
Comment (AlexSlawinski) [inline _includes/pbc/all/install-features/202400.0/install-the-service-points-order-management-feature.md:18]:
```suggestion
## Set up behavior
```

File: _includes/pbc/all/install-features/202400.0/install-the-service-points-order-management-feature.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+1.  Add a product offer with the service point shipment type to your cart.
+
+2.  Place an order with the added product.
+
+3.  Check that the `spy_sales_order_item_service_point` database table contains a record with the product and selected service point.
+
+{% endinfo_block %}
+
+## Install feature frontend
+
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202400.0/install-the-service-points-order-management-feature.md:75]:
```suggestion

Follow the steps below to install the {Feature Name} feature frontend.

```

File: _includes/pbc/all/install-features/202400.0/install-the-service-points-order-management-feature.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+
+This document describes how to integrate the Service Points + Order Management feature into a Spryker project.
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202400.0/install-the-service-points-order-management-feature.md:3]:
```suggestion
This document describes how to integrate the Service Points + [Order Management](/docs/pbc/all/order-management-system/{{page.version}}/base-shop/order-management-feature-overview/order-management-feature-overview.html) feature into a Spryker project.
```

**Review by VadymSachenko — APPROVED** [review]
Added IG to the Order Management PBC. Approved.
